nodes/vr_ros/vr-ros.go

```diff
@@ -84,3 +91,83 @@ func (n *vrRos) Init(cfg *types.NodeConfig, opts ...nodes.NodeOption) error {
 
 	return nil
 }
+
+// SaveConfig overrides the default VRNode SaveConfig to handle MikroTik RouterOS
+// Uses direct SSH connection since scrapligo doesn't support mikrotik_routeros platform.
+func (n *vrRos) SaveConfig(_ context.Context) error {
+	// Create SSH client configuration
+	config := &ssh.ClientConfig{
+		User: n.Credentials.GetUsername(),
+		Auth: []ssh.AuthMethod{
```

Comment:
@AAm-kun thanks
we will have to refactor this to use scrapligo's generic driver to abstract away low level ssh connection handling. If you [check out how scrapligo uses the GenericDriver](https://github.com/scrapli/scrapligo/tree/main/examples/generic_driver/basicshttps://github.com/scrapli/scrapligo/tree/main/examples/generic_driver/basics) you could probably refactor your existing PR with using it instead.